tests/network/connectivity/test_pod_network.py

```diff
@@ -100,6 +100,7 @@ def cloud_init_ipv6_network_data(dual_stack_network_data):
 @pytest.mark.gating
 @pytest.mark.single_nic
 @pytest.mark.s390x
+# Not marked as `conformance`; requires NMState
```

Comment:
This is a pod-network test, it shouldn't require using nmstate.

Reply:
but it should not block this PR